libsysbox/sysbox/sysbox.go

```diff
@@ -42,9 +43,18 @@ var uidShiftDistros = []string{"ubuntu"}
 // running containers inside a system container.
 func checkUnprivilegedUserns() error {
 
-	// Debian & Ubuntu
-	path := "/proc/sys/kernel/unprivileged_userns_clone"
-	if _, err := os.Stat(path); err == nil {
+	distro, err := libutils.GetDistro()
+	if err != nil {
+		return err
+	}
+
+	switch distro {
+
+	case "debian", "ubuntu":
+		path := "/proc/sys/kernel/unprivileged_userns_clone"
+		if _, err := os.Stat(path); err != nil {
```

Comment:
Do we need to `stat()`? Why not just open the file and check if that fails.

Reply:
Yep, makes sense, in that case I would need to change the existing code too ('ubuntu' case), I was just trying to be consistent here. Will change.